packages/components/MenuButton/macos/FRNMenuButtonManager.m

```diff
@@ -3,13 +3,14 @@
 
 @implementation RCTConvert (FRNMenuButtonAdditions)
 
-+ (NSMenuItem *)menuItem:(id)json
++ (NSMenuItem *)NSMenuItem:(id)json
```

Comment:
for my own knowledge why to NSMenuItem?

Reply:
The method names of RCTConvert usually match the object it returns. I also made it so that the JSON object passed in has keys that match the properties on NSMenuItem. So prior to this change, [RCTConvert menuItem:] took in JSON that was pretty specific to FURN MenuButton. After this change, it takes in a JSON that is much more generic, so I felt the name change appropriate.

I changed this method to be more generic in preparation for when I pull it out of MenuButton and into a standalone ContextMenu component.